omero/sysadmins/limitations.rst

```diff
@@ -43,15 +43,28 @@ user, you cannot make the group into a private group.
 File format support
 -------------------
 
+Large images
+^^^^^^^^^^^^
+
+When you import an image over a certain size, OMERO will generate a Pyramid of lower resolution
+images if it doesn't already exist in the file. The threshold size is configurable using
+:property:`omero.pixeldata.max_plane_height` and
+:property:`omero.pixeldata.max_plane_width` but set to 3192x3192 pixels by
+default. However, this process can be very resource-intensive for larger images, for example see
```

Comment:
The resource-intensiveness largely depends on a set of factor. Size is one of them but compression/decompression and optionally concurrency of multiple generation. I would not be too specific about the cause